packages/semantic-tokens/src/fluentOverrides.ts

```diff
@@ -170,10 +174,40 @@ export const fluentOverrides: FluentOverrides = {
   sizeCtrlLgIcon: { rawValue: '24px' },
   sizeCtrlSmDefault: { rawValue: '32px' },
   sizeCtrlSmIcon: { rawValue: '20px' },
+  statusBrandBackground: { f2Token: 'colorBrandBackground' },
+  statusBrandForeground: { f2Token: 'colorNeutralForegroundOnBrand' },
+  statusBrandTintBackground: {f2Token: 'colorBrandBackground2'},
+  statusBrandTintForeground: { f2Token: 'colorBrandForeground1' },
+  statusBrandTintStroke: { f2Token: 'colorBrandStroke2' },
   statusDangerBackground: { f2Token: 'colorPaletteRedBackground3' },
+  statusDangerForeground: { f2Token: 'colorNeutralForegroundOnBrand' },
+  statusDangerStroke: { f2Token: 'colorPaletteRedBorder2' },
+  statusDangerTintBackground: { f2Token: 'colorPaletteRedBackground1' },
   statusDangerTintForeground: { f2Token: 'colorPaletteRedForeground3' },
+  statusDangerTintStroke: { f2Token: 'colorPaletteRedBorder1' },
+  statusImportantBackground: {f2Token: 'colorNeutralForeground1' },
+  statusImportantForeground: {f2Token: 'colorNeutralBackground1' },
+  statusImportantStroke: { f2Token: 'colorNeutralStrokeAccessible' },
+  statusImportantTintBackground: { f2Token: 'colorNeutralForeground3' },
+  statusImportantTintForeground: { f2Token: 'colorNeutralForeground1' },
+  statusImportantTintStroke: { f2Token: 'colorTransparentStroke' },
+  statusInformativeBackground: {f2Token: 'colorNeutralBackground5' },
+  statusInformativeForeground: {f2Token: 'colorNeutralForeground3' },
+  statusInformativeStroke: { f2Token: 'colorNeutralStroke2' },
+  statusInformativeTintBackground: { f2Token: 'colorNeutralBackground4' },
+  statusInformativeTintForeground: { f2Token: 'colorNeutralForeground3' },
+  statusInformativeTintStroke: { f2Token: 'colorNeutralStroke2' },
   statusSuccessBackground: { f2Token: 'colorPaletteGreenBackground3' },
-  statusWarningBackground: { f2Token: 'colorPaletteDarkOrangeBackground3' },
```

Comment:
We need to be pretty careful about changing these values - this token is already used elsewhere (progress-bar) so overriding it will cause that component to break - we should add a variant token for statusWarningBackground in 'fluentLegacyVariant.ts' so that they can both have their own fallback opinion.